deploying-an-existing-application-using-okteto-ui.md

```diff
@@ -0,0 +1,175 @@
+# Deploying An Existing Application Using Okteto UI
+
+In the previous blog post, you learned how to build a FastAPI CRUD application and deployed it to Okteto using the Okteto CLI tool. In this tutorial, you will be learning how to deploy the application directly from your Okteto dashboard.
+
+## Initial Setup
+
+Start by creating a fork of the application from [GitHub](https://github.com/okteto/fastapi-crud) and then clone it locally. In your local clone, set it up using the following commands:
+
+```bash
+$ cd fastapi-crud
+$ python3 -m venv venv && source venv/bin/activate
+(venv)$ pip install -r requirements.txt
+```
+
+Verify that the setup is complete by running:
+
+```bash
+python3 main.py
+```
+
+## Initial Deployment
+
+In the previous post, you defined an Okteto stack file to facilitate the deployment of your application from the CLI tool. In this section, you will be deploying the application from the dashboard.
+
+However, before deployment, you need to reassign the repository where Okteto will store your application images.
+
+Your current `okteto-stack.yaml` file is similar to the code below:
+
+
+```yaml
+name: fastapi-crud
+services:
+  fastapi:
+    public: true
+    image: DOCKERHUBUSERNAME/fastapi-crud:latest
+    build: .
+    replicas: 1
+    ports:
+      - 8080
+    resources:
+      cpu: 100m
+      memory: 128Mi
+```
+
+On line 5, modify the `image` value from:
+
+```yaml
+image: DOCKERHUBUSERNAME/fastapi-crud:latest
+```
+
+to: 
+
+```yaml
+image: okteto.dev/fastapi-crud:latest
+```
+
+In the code block above, you renamed the storage location of your application's image from your dockerhub to Okteto's repository.
+
+### What's the Okteto.dev repository ?
+
+The `okteto.dev` repository is the central repository for images of applications deployed from Okteto's UI. The need for the reassignment of the image repository in the stack file is because Okteto lacks the permission to push images into your docker repository from the dashboard as opposed to when deployed from the CLI, where you are required to have an active docker session on your machine.
```

Comment:
the registry works with both UI and CLI. Instead of this description, I'd like us to showcase it something like this:


```suggestion
The `okteto.dev` repository is a container registry hosted in Okteto Cloud that allows every user to have its own space to store its container images. It's fully integrated with the rest of Okteto. In the previous post, we were using Docker Hub. Here, we are switching to Okteto's so our application deploys faster, and with fewer external dependencies.
```